cookbooks/aws-parallelcluster-scheduler-plugin/recipes/init.rb

```diff
@@ -18,6 +18,14 @@
 # create system user
 include_recipe "aws-parallelcluster-scheduler-plugin::init_user"
 
+# Add scheduler plugin virtualenv to default path
+template "/etc/profile.d/scheduler_plugin_virtualenv.sh" do
```

Comment:
this is going to be set for all the users in this way.
Instead, this must be set only for the scheduler plugin user, let's find a way to set it only in the scheduler plugin user profile